certification/pyxis/pyxis.go

```diff
@@ -122,7 +120,7 @@ func (p *pyxisEngine) createRPMManifest(ctx context.Context, rpmManifest *RPMMan
 }
 
 func (p *pyxisEngine) GetProject(ctx context.Context) (*CertProject, error) {
-	req, err := newRequestWithApiToken(ctx, http.MethodGet, getPyxisUrl(fmt.Sprintf("projects/certification/id/%s", p.ProjectId)), p.ApiToken)
+	req, err := p.newRequestWithApiToken(ctx, http.MethodGet, getPyxisUrl(fmt.Sprintf("projects/certification/id/%s", p.ProjectId)), nil)
```

Comment:
Is this `nil` by design?

Reply:
Yes. It signifies that there is no body.